app/controllers/projects_controller.rb

```diff
@@ -93,8 +93,8 @@ def show
     @project_metadata = @project.metadata
     @project_id = @project_metadata[:project_id] || {}
     @storage_capacity = @project_metadata[:storage_capacity]
-    @size = @storage_capacity[:size]
-    @unit = @storage_capacity[:unit]
+    @size = @storage_capacity[:size] || 0 # TODO: What should happen if @storage_capacity is nil? 
```

Comment:
@bess I doubt this will solve the error you are seeing.  The error indicates that the @storage_capacity is nil, not the values in the hash